src/Tests/Framework/ManagedElasticsearch/Tasks/InstallationTasks/DownloadMachineLearningSampleDataDistribution.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.IO;
+using ICSharpCode.SharpZipLib.GZip;
+using ICSharpCode.SharpZipLib.Tar;
+using Tests.Framework.ManagedElasticsearch.Nodes;
+
+namespace Tests.Framework.ManagedElasticsearch.Tasks.InstallationTasks
+{
+	public class DownloadMachineLearningSampleDataDistribution : InstallationTaskBase
```

Comment:
Apppears to be unused could be related to commented code in `MachineLearningCluster`

Reply:
Yup, fixed and retained.